src/libraries/System.Data.Common/src/System/Data/Common/DbDataAdapter.cs

```diff
@@ -398,6 +406,7 @@ protected virtual DataTable[] FillSchema(DataSet dataSet, SchemaType schemaType,
             }
         }
 
+        [RequiresUnreferencedCode(DataSet.RequiresUnreferencedCodeMessage)]
```

Comment:
What is RUC about this code? I'm not seeing what is unsafe. If we can remove this attribute, I think the ctor of this class wouldn't need to be marked RUC.